Tests/BootCamp.Chapter.Tests/HandTest.cs

```diff
@@ -10,7 +10,18 @@ public class HandTest
     {
         // The constructor cannot be tested because cards is protected. 
 
-        // AddCards cannot be tested because the return type is void and Cards cannot be read outside the class. 
+        [Fact]
+        public void AddCards_Given_Empty_List_And_A_List_With_Elements_Returns_A_List_With_Elements()
```

Comment:
*Given* is for preconditions
*When* is for args

In your case:
`AddCards_Given_NoCards_When_SingleCard_Returns_Deck_With_SingleCard`

Reply:
Read the test to your wife. If she cannot understand it- the test is not being named well.